GetARock/GetARock/Global/UIComponent/BasicTextView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  BasicTextView.swift
+//  GetARock
+//
+//  Created by Jisu Jang on 2023/01/25.
+//
+import UIKit
+
+final class BasicTextView: UIView {
+
+    private let placeholder: String
+
+    private var maxCount: Int = 300
+
+    private lazy var textView: UITextView = {
+        // 행간 세팅
+        let style = NSMutableParagraphStyle()
+        let attributedString = NSMutableAttributedString(string: $0.text)
+        style.lineSpacing = CGFloat(10)
+        attributedString.addAttribute(NSAttributedString.Key.paragraphStyle, value: style, range: NSRange(location: 0, length: attributedString.length))
+
+        $0.attributedText = attributedString
+        $0.font = UIFont.setFont(.content)
+        $0.textColor = .white
+        $0.backgroundColor = .dark02
+
+        $0.delegate = self
+        return $0
+    }(UITextView())
+
+    private lazy var countLabel = BasicLabel(contentText: "\(textView.text.count)/\(maxCount)", fontStyle: .content, textColorInfo: .gray02)
```

Comment:
p2;
코드 전체적으로 120자가 넘어가는 코드가 많아요..!
컨벤션에 따라 개행 부탁드립니다!

```suggestion
    private lazy var countLabel = BasicLabel(
        contentText: "\(textView.text.count)/\(maxCount)",
        fontStyle: .content,
        textColorInfo: .gray02
    )
```

Reply:
반영했습니다! [546abe8](https://github.com/extreme-rock/GetARock-iOS/pull/49/commits/546abe8167c3ea9fafbfde0f2a4cb6d26b88caee)

120자를 지키니 확실히 더 깔끔해진 거 같아요 담부터 더 주의해서 지켜야겟습니다!! 🔥